src/main/kotlin/no/nav/helse/dokument/K9MellomlagringGateway.kt

```diff
@@ -90,18 +90,22 @@ class K9MellomlagringGateway(
     }
 
     internal suspend fun slettDokmenter(
-        urls: List<URI>,
+        vedleggId: List<String>,
```

Comment:
Fint om du kaller det for dokumentId. Det er ikke bare vedlegg vi oppretter eller slettet :)